contributing/documentation/overview.rst

```diff
@@ -263,11 +263,13 @@ changes.
 To access the `Platform.sh`_ environment URL, simply go to your Pull Request 
 page on GitHub and click on ``Details``.
 
-.. note::
+The specific configuration files at the root of the Git repository: 
+``.platform.app.yaml``, ``.platform/services.yaml`` and ``.platform/routes.yaml`` 
+allow `Platform.sh`_ to build Pull Requests.
 
-    The specific configuration files at the root of the Git repository: 
-    ``.platform.app.yaml``, ``.platform/services.yaml`` and 
-    ``.platform/routes.yaml`` allow `Platform.sh`_ to build Pull Requests.
+.. note::
+    Only the following branches are supported for building Pull Requests: 2.3 (LTS), 
+    2.6, 2.7, 2.8.
```

Comment:
I think we should just state here that we only maintain the configuration for maintained branches (which correspond to the currently maintained Symfony versions). This way we would save us work when versions reach end of support and so on.